packages/extras/src/lib/types/components.ts

```diff
@@ -94,6 +94,6 @@ export type TextProperties = Omit<MeshInstanceProperties, 'mesh'> & {
 
 export type GLTFProperties = Omit<Object3DInstanceProperties, 'object'> & {
   url: string
-  dracoDecoderPath?: string
+  dracoDecoderPath?: string | boolean
```

Comment:
I think we should align the API of `useGltf` and `<GLTF>` so that this property should be called `useDraco`. The "use" also implies that a boolean value can be assigned.

Reply:
Should I just add a new `useDraco` and add a deprecation note for the older `dracoDecoderPath` so that it warns devs to use the newer property?
With that, the older `dracoDecoderPath` would be removed by v5.0.0 
What do you think?

Reply:
Maybe something like 
```js
if (dracoDecoderPath) { // it would be reverted back to the current type (string|undefined)
  console.warn('dracoDecoderPath will be deprecated soon, please use useDraco instead');
}
```

Reply:
I just pushed my proposed changes, but if you don't like it @grischaerbe , it's Ok, I'll remove them and do exactly what you said.